src/Tizen.NUI.Physics2D/src/public/chipmunk/BoundingBox.cs

```diff
@@ -111,16 +111,16 @@ public override bool Equals(object obj)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override int GetHashCode()
         {
-            var hashCode = -1064806749;
-
+            unchecked
+            {
 #pragma warning disable RECS0025 // Non-readonly field referenced in 'GetHashCode()'
-            hashCode = hashCode * -1521134295 + left.GetHashCode();
-            hashCode = hashCode * -1521134295 + bottom.GetHashCode();
-            hashCode = hashCode * -1521134295 + right.GetHashCode();
-            hashCode = hashCode * -1521134295 + top.GetHashCode();
+                int hash = left.GetHashCode();
+                hash = (hash * 7) + bottom.GetHashCode();
+                hash = (hash * 7) + right.GetHashCode();
+                hash = (hash * 7) + top.GetHashCode();
 #pragma warning restore RECS0025 // Non-readonly field referenced in 'GetHashCode()'
-
-            return hashCode;
+                return hash;
```

Comment:
It would change hash algorithm.
Please check if System.Numerics.BigInteger can be used here.